tests/test_performance.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""Performance tests.
+
+These tests train models on larger datasets, and might take a long time.
+"""
+
+import os
+import unittest
+
+import pytest
+
+from pykeen.experiments.cli import HERE
+from pykeen.pipeline import pipeline_from_path
+
+
+@pytest.mark.slow
+class RGCNTests(unittest.TestCase):
```

Comment:
this one is realllllly slow on CPU (mine claims it will take 4 hours), so it can't really live in the testing pipeline that gets run with unittests on travis

Reply:
@cthoyt This test trains one of the reproducibility configurations. Thus, it is expected to take very long. I would not recommend having this as part of the CLI, but found it convenient for myself to have this check as a test which I can trigger manually.

Reply:
@mberr how is this looking now?